drivers/led/lp50xx.c

```diff
@@ -39,7 +39,7 @@ LOG_MODULE_REGISTER(lp50xx, CONFIG_LED_LOG_LEVEL);
 
 /* Maximum number of channels */
 #define LP50XX_MAX_CHANNELS(nmodules)	\
-	((LP50XX_COLORS_PER_LED + 1) * ((nmodules) + 1))
+	(LP50XX_COLORS_PER_LED * nmodules)
```

Comment:
Please can you explain what you are fixing ?

I find this change highly suspicious. For each module, there are three color registers and one brightness register, so `LP50XX_COLORS_PER_LED + 1` makes sense. And the ->write_channels() API also exposes the bank registers. And that's `(nmodules) + 1`.

So I don't understand :)